src/global/app-css.scss

```diff
@@ -9,11 +10,60 @@
 @import "tokens/screens.scss";
 @import "tokens/sizes.scss";
 
+/**
+ * Minimal CSS Reset
+ */
+
 html {
+  box-sizing: border-box;
   -webkit-font-smoothing: antialiased;
 }
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  overflow-wrap: break-word;
+}
+
+p, ul, ol, blockquote, figure {
+  overflow-wrap: break-word;
+}
+
+:where(a) {
```

Comment:
Curious how this is different from just `a {`?

Reply:
Yeah, so `:where` uses very low specificity, which means it's easy to override regardless of source order. Good for "resets" and defaults.